TrafficCapture/dockerSolution/src/main/docker/otelCollector/otel-config-aws.yaml

```diff
@@ -3,6 +3,22 @@ receivers:
     protocols:
       grpc:
 processors:
+  cumulativetodelta:
+  probabilistic_sampler/traces:
+    sampling_percentage: ${env:TRACE_SAMPLING_PERCENTAGE:-1}
+  resource/metrics:
+    attributes:
+      - key: qualifier
+        value: ${env:QUALIFIER:-unknown}
```

Comment:
Do we expect QUALIFIER to ever be absent?  "unknown" might be confusing for users.